api/src/uvisor-input.S

```diff
@@ -93,14 +91,23 @@ uvisor_config:
     /* Privileged system IRQ hooks */
     .long __uvisor_priv_sys_irq_hooks
 
+/* uVisor mode of operation
+ * 0: Disabled
+ * 1: Permissive
+ * 2: Enabled */
```

Comment:
I've fine having the comment use the `#define` names. Those are part of our API can shouldn't change.

I'm also fine removing the comment, but there must be some value in having the comment as you felt there was a reason to add it.